Cabal/Distribution/PackageDescription/Parse.hs

```diff
@@ -161,6 +162,14 @@ storeXFieldsPD _ _ = Nothing
 -- ---------------------------------------------------------------------------
 -- The Library type
 
+subLibFieldDescrs :: [FieldDescr Library]
+subLibFieldDescrs =
+  [ simpleField "library"
+                        showToken             parseTokenQ
+                        (fromMaybe "subLibFieldDescrs" . libName)
```

Comment:
Looks like used only for pretty printing? I'd make `parseTokenQ` and `(from ..` calls to `cabalBug`, as this is not intended for parsing.

Also would leave a comment: `PackageDescription` should be printed using *section syntax*. Will it explain, why there is `executable` field in the `executable` ?